internal/rbd/rbd_journal.go

```diff
@@ -530,10 +530,13 @@ func undoVolReservation(ctx context.Context, rbdVol *rbdVolume, cr *util.Credent
 // Generate new volume Handler
 // The volume handler won't remain same as its contains poolID,clusterID etc
 // which are not same across clusters.
+// nolint:gocyclo,nestif,cyclop // reduce complexity
```

Comment:
avoid adding this one? try refractoring

Reply:
Yes, agree. as I mentioned earlier sadly it's not recomended to add nolint , max we try to avoid adding it because if we do so no limit for adding more complexity